### ISSUE TITLE
Sharing folders not being discovered

### ISSUE THREAD

**Issue by radicalhashcan** [slskd/slskd#200]
Currently sharing a folder via an E: drive similar to this config layout

 directories:
   app: ~
   incomplete: ~
   downloads: e:\
   shared: E:Music

It's however unable to find the files/shared folders any idea why?

**Comment by jpdillingham** [comment]
Try:

```yaml
directories:
  shared:
    - 'E:\Music'
```

The `shared` configuration is expected to be a list/array as of [0.13.0](https://github.com/slskd/slskd/releases/tag/0.13.0)

**Comment by radicalhashcan** [comment]
yeah understood a bit confusing to a new user might wanna add more comments giving examples within the default configuration was just assuming it was same because setting the download folder worked just fine. Thanks again as always

**Comment by radicalhashcan** [comment]
Gonna reopen this issue as I am testing downloading from a nicotine instance I am running into an upload rejection issue. Specifically the file not being found. Again this is confusing because the software has reported has finding 8000 files is this due to name scheming?

**Comment by radicalhashcan** [comment]
[UPLOAD REJECTED] File @@xxoMMusic\Trippie Redd - Trip At Knight (Complete Edition) (2021) - WEB FLAC\01. Molly Hearts.flac not found.

Is an example of output

**Comment by jpdillingham** [comment]
At first glance it looks like the path alias/mask is getting mangled.  I can't recall if I tested this with a colon in the path, but I'll do that this evening.

**Comment by jpdillingham** [comment]
Confirmed that it's a bug.

**Comment by jpdillingham** [comment]
The canary should be finished building about 20 minutes from now, please give it a shot and let me know if this fixes the problem.

**Comment by jpdillingham** [comment]
@radicalhashcan I'm going to go ahead and close this, feel free to re-open it if you're still having problems.

**Comment by jpdillingham** [comment]
@RandomTypek I'll try to reproduce this.  In the meantime, could you do me a favor and check to see if it works with a path that only contains lowercase letters?

**Comment by jpdillingham** [comment]
I'll close this issue for now, but due to the amount of unanticipated behavior coming from the static methods in `Path` from the .NET framework, a complete overhaul of path manipulation logic might be warranted to really solve these issues once and for all.
